image-index.md

```diff
@@ -93,7 +93,7 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
 - **`subject`** *[descriptor](descriptor.md)*
 
     This OPTIONAL property specifies a [descriptor](descriptor.md) of another manifest.
-    This value, used by the [`referrers` API][referrers-api], indicates a relationship to the specified manifest.
+    This value defines a loose relationship to a separate DAG structure, and is used by the [`referrers` API][referrers-api] to this manifest in the list of responses for the subject digest.
```

Comment:
"a weak association"?

^ to be more formal

Reply:
If you are referencing a DAG, then maybe you can qualify it similar to 
https://github.com/opencontainers/image-spec/blob/970322241ad9c8868d15e6859c9b446ad06b944d/descriptor.md?plain=1#L3

Reply:
Both suggestions applied. Thanks!